scripts/ci/docker-compose/base.yml

```diff
@@ -78,16 +78,35 @@ services:
       - VERSION_SUFFIX_FOR_PYPI=${VERSION_SUFFIX_FOR_PYPI}
       - VERSION_SUFFIX_FOR_SVN=${VERSION_SUFFIX_FOR_SVN}
       - WHEEL_VERSION=${WHEEL_VERSION}
+      - OTEL_TRACES_EXPORTER=jaeger_thrift
```

Comment:
Those variables should only be set when "open-telemetry" `integration` is enabled. See the following comments.